game.rb

```diff
@@ -0,0 +1,50 @@
+require_relative 'environment'
+
+class Game 
+  attr_accessor :depot, :control
+  include Environment
+
+  def initialize
+    @board = { payers: {}, control: {}, status: nil}
```

Comment:
An extra space at the end of the line is needed. [Rubocop](https://docs.rubocop.org/rubocop/1.23/index.html) gem can complain in a real life project.